lib/alchemy/upgrader/four_point_zero.rb

```diff
@@ -1,3 +1,68 @@
+require 'thor'
+
+class Alchemy::Upgrader::FourPointZeroTasks < Thor
+  include Thor::Actions
+
+  no_tasks do
+
+    def convert_available_contents_to_nestable_elements
```

Comment:
Metrics/AbcSize: Assignment Branch Condition size for convert_available_contents_to_nestable_elements is too high. [34.23/15]